swap_meet/item.py

```diff
@@ -1,2 +1,24 @@
+import uuid
+
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, id = None, condition = 0):
+        self.id = uuid.uuid1().int if id is None else id
+        self.condition = condition
+
+    def get_category(self):
+        return __class__.__name__
```

Comment:
Here you correctly return the class name, but you're not grabbing the class name from `self`. Since `get_category` is an instance method and not a class method, I think it would be better suited to grab the class name from `self` directly like: `return self.__class__.__name__`